src/main/daml/Daml/Finance/Lifecycle/Rule/Claim.daml

```diff
@@ -90,6 +92,10 @@ template Rule
           description = effectView.description
           contextId = Some effectView.id
           routedSteps
-          settlementTime = Some $ time effectView.settlementDate 0 0 0
+          oSettlementTime =
```

Comment:
Let's discuss during the team meeting, but it's probably worth changing the `effect` implementation to also have `Time` instead of `Date`.